### PR TITLE
Cleanup 🗑️

### PR DESCRIPTION
**Pull request by Windvis**
A bunch of small changes to remove unused or unneeded things.

### CODE REVIEW

File: config/environment.js [hunk 1]
```diff
@@ -28,12 +16,7 @@ module.exports = function (environment) {
     },
   };
 
-  if (environment === 'production') {
-    ENV.yappConfigKey = 'dev';
```
Comment (Windvis) [inline config/environment.js:32]: No idea what this is / was.

File: config/environment.js [hunk 2]
```diff
@@ -28,12 +16,7 @@ module.exports = function (environment) {
...
-    ENV.API_URL = 'https://burgernabije-besluitendatabank-dev.s.redhost.be/api';
```
Comment (Windvis) [inline config/environment.js:33]: API_URL wasn't used anywhere anymore.

File: app/components/vote-overview.ts [hunk 3]
```diff
@@ -1,6 +1,4 @@
 import Component from '@glimmer/component';
-import { tracked } from '@glimmer/tracking';
-import * as d3 from 'd3';
```
Comment (Windvis) [inline app/components/vote-overview.ts:3]: This one has the most impact on the app. Now only the map route imports d3 so Embroider can move it into the map route bundle which means the main bundle is a little bit smaller again 🎉.

**Review by aliokan — APPROVED** [review]
Great! :pray:

**Pull request merged**